source/components/GlobalNavigation/components/Search/Search.js

```diff
@@ -354,37 +361,40 @@ class Search extends React.Component {
                     onClick={this.onSearchClose}
                     text
                 >
-                    <Icon name="close" className="wds-global-navigation__search-close-icon" tiny />
+                    <Icon name="close-tiny" className="wds-global-navigation__search-close-icon" tiny />
```

Comment:
`Icon` is deprecated, right? Maybe swap this with `@wikia/react-common/icons/IconCloseTiny`?

Reply:
at the moment of creation, there were issues with `IconXXX` components which are now resolved. I'll go through the whole GN codebase and update it in the last PR